src/scenes/Organizations.re

```diff
@@ -7,74 +9,222 @@ module Styles = {
+    let%Hook snackbar =
+      Snackbar.make(
+        ~action={
+          Paper.Snackbar.snackbarAction(~label="Close", ~onPress=ignore);
+        },
+      );
+
+    let%Epitath createOrganization = c =>
+      <CreateOrganizationMutationContainer>
+        ...{(mutate, _) => c(mutate)}
+      </CreateOrganizationMutationContainer>;
+
+    <StackNavigator.Screen navigation>
+      ...{() =>
+        <QueryRenderer result refetch>
+          ...{data =>
+            <View style=Styles.wrapper>
+              <AppStatusBar mode=Light />
+              <AppBar
+                title="Settings"
+                renderLeft={_ => <BackButton onPress=ignore />}
+              />
+              <SectionTitle
+                value="Organizations"
+                style=Style.(style([paddingHorizontal(Pt(20.))]))
+              />
+              <AppList>
+                {switch (result) {
+                 | Error(_) => <Whoops />
+                 | Data(data) =>
+                   data
+                   ->Belt.Option.flatMap(data =>
+                       data##currentUser##organizations##edges
+                     )
+                   ->Belt.Option.getWithDefault([||])
+                   ->(
+                       organizations =>
+                         renderOrganizations(~fetchMore, ~data, organizations)
+                     )
+                 | Loading =>
+                   <View style=Styles.notFoundContentWrapper>
+                     <FullLoader mode=Black />
+                   </View>
+                 }}
+              </AppList>
+              <AppFAB
+                icon={
+                  <RNIcons.MaterialCommunityIcons
+                    name=`_plus
+                    size=24.
+                    color={AppConfig.theme.contrastWhite}
+                  />
+                }
+                onPress={() =>
+                  Js.Promise.(
+                    createOrganization()
+                    |> then_(result =>
+                         CreateOrganizationMutationContainer.convertJsInputToReason(
+                           result,
+                         ).
+                           data
+                         ->Belt.Option.map(data => data##createOrganization)
+                         ->Belt.Option.map(createOrganization =>
+                             switch (createOrganization##error) {
+                             | Some(error) =>
+                               snackbar.send(Set(Open(error)));
+                               Js.Promise.resolve();
+                             | _ =>
+                               snackbar.send(
+                                 Set(Open("Organization Created")),
+                               );
+                               Js.Promise.resolve();
+                             }
+                           )
+                         ->Belt.Option.getWithDefault(Js.Promise.resolve())
+                       )
+                    |> catch(_ => resolve())
```

Comment:
should show an error to the user here too!

Reply:
I think I indeed did like this in some other places in some projects, need to fix 🤔